### DISCUSSION TITLE
Will all the DNS queries from AdGuard Home go out encrypted by default?

### DISCUSSION THREAD

**Discussion by logicito** [AdguardTeam/AdGuardHome#7162] [General]
When my client devices make DNS requests, they go to AdGuard Home within the OpenWrt router (both together in the same device), will those DNS queries go out to the internet (Quad9.net) encrypted via HTTPS or TLS? Thanks

**Comment by Dynamic5912** [comment]
> When my client devices make DNS requests, they go to AdGuard Home within the OpenWrt router (both together in the same device), will those DNS queries go out to the internet (Quad9.net) encrypted via HTTPS or TLS? Thanks

What DNS servers have you set for Quad9?
